hack/test-extended/builds/run.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
Lines 1-43 here could be their own function, I can't see that start-up sequence changing for different buckets.